Description: Upgrade ptr_meta to 0.3 for rancor
 .. bytecheck itself would need an update to 0.8, but that involves multiple
 dependents, complicating the matter.
Forwarded: not-needed
Last-Update: 2025-01-15
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -34,3 +34,3 @@
 [dependencies.ptr_meta]
-version = "0.1"
+version = "0.3"
 default-features = false
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -163,3 +163,3 @@
 };
-use ptr_meta::PtrExt;
+use ptr_meta as PtrExt;
 #[cfg(all(feature = "simdutf8", not(feature = "verbose")))]
